features/config/upgrade/src/test/java/org/opennms/config/upgrade/LiquibaseUpgraderIT.java

```diff
@@ -189,7 +184,9 @@ public void shouldRunChangelog() throws LiquibaseException, IOException, SQLExce
             // check if schema changes work properly
             Optional<ConfigDefinition> configDefinition = this.cm.getRegisteredConfigDefinition(SCHEMA_NAME_PROPERTIES);
             assertTrue(configDefinition.isPresent());
-            ConfigItem schema = configDefinition.get().getSchema();
+
+            OpenAPI openApi = configDefinition.get().getSchema();
+            ConfigItem schema = OpenAPIBuilder.createBuilder(SCHEMA_NAME_PROPERTIES, SCHEMA_NAME_PROPERTIES, "", openApi).getRootConfig();
             assertNotNull(schema);
             assertEquals(2, schema.getChildren().size());
             assertEquals("property1", schema.getChildren().get(0).getName());
```

Comment:
@freddy61025 this fails.
It seems that the name of a property gets lost while serializing into the database?